apps/src/gamelab/GameLabP5.js

```diff
@@ -456,9 +456,9 @@ GameLabP5.prototype.startExecution = function () {
           }
 
           this._setProperty('frameCount', this.frameCount + 1);
+          this.redraw();
           this._updateMouseCoords();
           this._updateTouchCoords();
-          this.redraw();
```

Comment:
See matching (correct) sequence in [p5.js' core.js `_draw()` function](https://github.com/processing/p5.js/blob/0.5.4/src/core/core.js#L356).